spaces/domains.json

```
@@ -1,4 +1,4 @@
-{
+
```

You should not remove the brackets that wrap the list.
```suggestion
{
```